Sources/PostgresNIO/New/Messages/ErrorResponse.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 06.01.21.
+//
+
+extension PSQLBackendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLBackendMessage {
```